<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshDialog</class>
 <widget class="QDialog" name="MeshDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lVertex">
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbTable">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCalculate">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hlLeftPbCalculate">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbWritingImage">
       <property name="text">
        <string>Write image</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="MeshGraphicsView" name="graphicsView">
     <property name="cacheMode">
      <set>QGraphicsView::CacheBackground</set>
     </property>
     <property name="viewportUpdateMode">
      <enum>QGraphicsView::FullViewportUpdate</enum>
     </property>
     <property name="optimizationFlags">
      <set>QGraphicsView::DontAdjustForAntialiasing|QGraphicsView::DontClipPainter|QGraphicsView::DontSavePainterState</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="lX">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>X : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lY">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Y : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lZ">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShowVertex">
       <property name="text">
        <string>Show vertex</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShowTriangles">
       <property name="text">
        <string>Show triangles</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShowImage">
       <property name="text">
        <string>Show image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbA">
       <property name="title">
        <string>Point A</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lAX">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>X : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lAY">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Y : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lAZ">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Z:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lAIndex">
          <property name="text">
           <string>Index :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbB">
       <property name="title">
        <string>Point B</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lBX">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>X : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lBY">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Y : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lBZ">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Z:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lBIndex">
          <property name="text">
           <string>Index :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbC">
       <property name="title">
        <string>Point C</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="lCX">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>X : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lCY">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Y : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lCZ">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Z:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lCIndex">
          <property name="text">
           <string>Index :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbTriangle">
       <property name="title">
        <string>Triangle</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="lTIndex">
          <property name="text">
           <string>Index :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lProcessName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MeshGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>MeshGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>MeshDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>728</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>663</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTable</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MeshDialog</receiver>
   <slot>selectTable(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowVertex</sender>
   <signal>toggled(bool)</signal>
   <receiver>MeshDialog</receiver>
   <slot>showVertex(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>921</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowTriangles</sender>
   <signal>toggled(bool)</signal>
   <receiver>MeshDialog</receiver>
   <slot>showTriangles(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>921</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbShowImage</sender>
   <signal>toggled(bool)</signal>
   <receiver>MeshDialog</receiver>
   <slot>showImage(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>921</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCalculate</sender>
   <signal>clicked()</signal>
   <receiver>MeshDialog</receiver>
   <slot>onPbCalculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbWritingImage</sender>
   <signal>clicked()</signal>
   <receiver>MeshDialog</receiver>
   <slot>onPbWriteImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>launchCompleteProcess()</slot>
  <slot>selectTable(QString)</slot>
  <slot>showVertex(bool)</slot>
  <slot>showTriangles(bool)</slot>
  <slot>showImage(bool)</slot>
  <slot>onPbCalculate()</slot>
  <slot>onPbWriteImage()</slot>
 </slots>
</ui>
